code/API_definitions/webrtc-registration.yaml

```
@@ -294,6 +294,8 @@ components:
       properties:
         deviceId:
           type: string
+          pattern: ^[a-zA-Z0-9-]{0,256}$
+          example: "7d444840-9dc0-11d1-b245-5ffdce74fad2"
           description: |-
             The deviceId of the client in UUID format. A unique identifier for
             the physical device where a registration is initiated. Generated
```

That pattern is unnecessarily complex as the API Provider can follow the UUID format, I would just recommend:

```suggestion
        deviceId:
          type: string
          format: uuid
          description: |-
            The deviceId of the client in UUID format. A unique identifier for
            the physical device where a registration is initiated. Generated
```